tests/src/test/resources/artemis/launch.sh

```diff
@@ -19,5 +19,7 @@
 #   cp $AMQ_HOME/conf/* ~/broker/etc/
 # .
 # This allows usage of a custom broker.xml and other config files, provided in the "$AMQ_HOME/conf/" directory.
-sed -i -E 's/^([[:space:]]*)configure[[:space:]]*$/\0\n\1cp $AMQ_HOME\/conf\/\* ~\/broker\/etc\//g' $AMQ_HOME/bin/launch.sh
-$AMQ_HOME/bin/launch.sh
+# We want the sed command to replace the actual occurrence of "cp $AMQ_HOME/conf/*"
```

Comment:
This sentence doesn't seem correct - see comment lines above (`replacing [...] with [...]`).